src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
+    {ok, Blacklist} = couch_util:parse_term(BlacklistConfig),
+    Key = binary_to_list(KeyBin),
+    lists:member(Key, Blacklist).
+
+ ensure_enabled(Key) ->
+    case is_disabled(Key) of
+      true ->
+        throw("The URL endpoint " ++ Key ++ "has been disabled.");
```

Comment:
Please select different value for error so it matches one of the clauses here:
https://github.com/apache/couchdb-chttpd/blob/master/src/chttpd.erl#L715
Possible options are:
- https://github.com/apache/couchdb-chttpd/blob/master/src/chttpd.erl#L729
- https://github.com/apache/couchdb-chttpd/blob/master/src/chttpd.erl#L737